roles/openshift_facts/library/openshift_facts.py

```diff
@@ -558,17 +597,17 @@ def set_deployment_facts_if_unset(facts):
             service_type = 'atomic-openshift'
             if deployment_type == 'origin':
                 service_type = 'origin'
-            elif deployment_type in ['enterprise', 'online']:
+            elif deployment_type in ['openshift-enterprise', 'enterprise', 'online']:
```

Comment:
These conditionals are for 3.0 deployments, we want the atomic-openshift service type for openshift-enterprise.

Likewise for /etc/origin and /var/lib/origin